### PR TITLE
docs/experimental-features.md: typo

### PR DESCRIPTION
**Pull request by RubenKelevra**

### CODE REVIEW

File: docs/experimental-features.md [hunk 1]
```diff
@@ -545,7 +544,7 @@ You can read more in the [datastore](./datastores.md#badgerds) documentation.
 ### State
 Experimental
 
-Allows to create directories with unlimited number of entries - currently
+Allows to create directories with an unlimited number of entries - currently
```
Comment (willscott) [inline docs/experimental-features.md]:
```suggestion
Allows creating directories with an unlimited number of entries - currently
```

File: docs/experimental-features.md [hunk 2]
```diff
@@ -571,20 +570,15 @@ ipfs config --json Experimental.ShardingEnabled true
 
 Experimental, default-disabled.
 
-Utilizes pubsub for publishing ipns records in real time.
+Utilizes pubsub for publishing ipns records in real-time.
```
Comment (willscott) [inline docs/experimental-features.md]:
```suggestion
Utilizes pubsub for publishing ipns records in real time.
```
in the noun form, two words is preferred: https://www.merriam-webster.com/dictionary/real%20time

File: docs/experimental-features.md [hunk 3]
```diff
@@ -571,20 +570,15 @@ ipfs config --json Experimental.ShardingEnabled true
...
 When it is enabled:
 - IPNS publishers push records to a name-specific pubsub topic,
   in addition to publishing to the DHT.
 - IPNS resolvers subscribe to the name-specific topic on first
-  resolution and receive subsequently published records through pubsub
-  in real time. This makes subsequent resolutions instant, as they
-  are resolved through the local cache. Note that the initial
-  resolution still goes through the DHT, as there is no message
-  history in pubsub.
+  resolution and receive subsequently published records through pubsub in real-time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.
```
Comment (willscott) [inline docs/experimental-features.md]:
```suggestion
  resolution and receive subsequently published records through pubsub in real time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.
```
https://www.merriam-webster.com/dictionary/real%20time

File: docs/experimental-features.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ run your daemon with the `--enable-pubsub-experiment` flag. Then use the
 
 Gossipsub is a new, experimental routing protocol for pubsub that
 should waste less bandwidth than floodsub, the current pubsub
-protocol. It's backwards compatible with floodsub so enabling this
```
Comment (Stebalien) [inline docs/experimental-features.md]: note: british english (but normalizing is fine)

File: docs/experimental-features.md [hunk 5]
```diff
@@ -617,14 +611,14 @@ Modify your ipfs config:
...
 ```
 
-For listening on a QUIC address, add it the swarm addresses, e.g. `/ip4/0.0.0.0/udp/4001/quic`.
+For listening on a QUIC address, add it to the swarm addresses, e.g. `/ip4/0.0.0.0/udp/4001/quic`.
 
 
 ### Road to being a real feature
 
-- [ ] The IETF QUIC specification needs to be finalised.
+- [ ] The IETF QUIC specification needs to be finalized.
```
Comment (Stebalien) [inline docs/experimental-features.md]: (also british english)

File: docs/experimental-features.md [hunk 6]
```diff
@@ -687,9 +681,9 @@ ipfs config --json Experimental.StrategicProviding true
 
 ### Road to being a real feature
 
-- [ ] needs real world testing
+- [ ] needs real-world testing
 - [ ] needs adoption
-- [ ] needs to support all providing features
+- [ ] needs to support all provided features
```
Comment (Stebalien) [inline docs/experimental-features.md]:
```suggestion
- [ ] needs to support all provider subsystem features
```

File: docs/experimental-features.md (hunk 4 above)
Comment (RubenKelevra) [reply docs/experimental-features.md:56]: yeah I know, should have added this to the changelog

**Pull request merged**